app/routes.js

```diff
@@ -6,7 +6,7 @@ import {
 } from "react-router";
 import App from "./components/App";
 import HomePage from "./components/home/HomePage";
-import UsersPage from "./components/users/UsersPage";
+import UsersPageComponent from "./components/users/UsersPage";
```

Comment:
I agree with this change, but should we change all referenced components so that we don't introduce an inconsistency?

Reply:
Nope.. please check my answer to Mati's question.